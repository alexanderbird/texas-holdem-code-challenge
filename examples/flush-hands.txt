2H 6H 8C 2C TH
Grant 7C 3C 9C
Brucie 4H 5H AH
Jewell AH 3H
Kalinda KH 4H
