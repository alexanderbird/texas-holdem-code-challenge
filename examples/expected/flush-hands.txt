2 Brucie Flush Ace
2 Jewell Flush Ace
3 Kalinda Flush King
4 Grant Flush 9
